akka-persistence-typed/src/main/scala/akka/persistence/typed/internal/ReplayingSnapshot.scala

```diff
@@ -179,7 +179,8 @@ private[akka] class ReplayingSnapshot[C, E, S](override val setup: BehaviorSetup
           System.nanoTime(),
           version,
           seenPerReplica,
-          eventsReplayed = 0))
+          eventsReplayed = 0,
+          metadata = None))
     }
 
```

Comment:
I don't store custom metadata with a snapshot. Do you see a need for that? It would then be accessible from OnRecoverComplete even if no events were replayed.

Reply:
I added it anyway since we already had `metadata: Option[Any]` in the same way in snapshots and it felt more complete, e.g. that you would see the same metadata from RecoverComplete if there were replayed events or just snapshot.
https://github.com/akka/akka/pull/32628/commits/0d7650cfafc3ac39d687c6556e1463fc28ec2353